docs/eventing/channels/create-default-channel.md

```diff
@@ -1,47 +1,68 @@
----
-title: "Creating a channel using cluster or namespace defaults"
-linktitle: "Creating channels"
-weight: 65
-type: "docs"
-showlandingtoc: "true"
----
-
 # Creating a channel using cluster or namespace defaults
 
-Developers can create channels of any supported implementation type by creating an instance of a Channel object.
+Developers can create channels of any supported implementation type by creating an instance of a
+Channel object.
+
+To create a channel:
+
+* Create a [Channel object](https://knative.dev/docs/reference/api/eventing/#messaging.knative.dev/v1.Channel) by running:
+
+    ```yaml
+    kubectl apply -f <<EOF
+    apiVersion: messaging.knative.dev/v1
+    kind: Channel
+    metadata:
+      name: <example-channel>
+      namespace: <namespace>
+    EOF
+    ```
+    Where:
+
+    * `<example-channel>` is the name of the channel you want to create.
+    * `<namespace>` is the name of your target namespace.
+
+If you create this object in the `default` namespace, according to the default ConfigMap
+example in [Channel types and defaults](/eventing/channels/channel-types-defaults), it is an InMemoryChannel
+channel implementation.
+
+<!-- TODO: Add tabs for kn etc-->
 
-1. Create a Channel object:
+After the Channel object is created, a mutating admission webhook sets the `spec.channelTemplate` based on the default channel implementation:
 
-      ```yaml
-      apiVersion: messaging.knative.dev/v1
-      kind: Channel
-      metadata:
-        name: my-channel
-        namespace: default
-      ```
+```yaml
+kubectl apply -f <<EOF
+apiVersion: messaging.knative.dev/v1
+kind: Channel
+metadata:
+  name: <example-channel>
+  namespace: <namespace>
+spec:
+  channelTemplate:
+    apiVersion: messaging.knative.dev/v1
+    kind: <channel-template-kind>
+EOF
+```
```

Comment:
I could be wrong, but I don't think this is something a user applies, I think it's supposed to just be an example of what the resource looks like after spec.channelTemplate is set?
Someone from @knative/eventing-reviewers can probably confirm better, cc @n3wscott

Reply:
Yeah, I changed it from a step to simply an introductory sentence.

Reply:
I just mean that I don't think this example needs ` kubectl apply -f <<EOF` etc because the changes we're talking about are applied by the webhook, not manually by the user - I think this one should just be an example of YAML showing the actual values rather than the variables etc. This would be worth double checking with some SME for eventing though

Reply:
Yes this correct @abrennan89  user doesn't create the object `spec:` this section just want to show what the controller will create, the user doesn't do the `kubectl` with the `channelTemplate` section.

Reply:
@RichardJJG if you can update on this one I can merge the PR